### PR TITLE
updating isOrphaned script

### PR DESCRIPTION
**Pull request by Bharath92**
https://github.com/Shippable/base/issues/1058

### CODE REVIEW

File: migrations/post_install/v5.2.4-post_install.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+do $$
+  begin
+    if exists (select 1 from information_schema.columns where table_name = 'projects' and column_name = 'isOrphaned') then
+      update projects set "isOrphaned" = false where "isOrphaned" is NULL;
```
Comment (Bharath92) [inline migrations/post_install/v5.2.4-post_install.sql]: 171.629 ms

File: migrations/post_install/v5.2.4-post_install.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      create temp table temp_pa as select id from projects;
```
Comment (Bharath92) [inline migrations/post_install/v5.2.4-post_install.sql]: 4.160 ms

File: migrations/post_install/v5.2.4-post_install.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+      delete from temp_pa using "projectAccounts" where "projectAccounts"."projectId" = temp_pa.id;
```
Comment (Bharath92) [inline migrations/post_install/v5.2.4-post_install.sql]: 17.218 ms

File: migrations/post_install/v5.2.4-post_install.sql [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+      update projects set "isOrphaned" = true from temp_pa where temp_pa.id = projects.id;
```
Comment (Bharath92) [inline migrations/post_install/v5.2.4-post_install.sql]: 29.507 ms

File: migrations/post_install/v5.2.4-post_install.sql [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+      drop table temp_pa;
```
Comment (Bharath92) [inline migrations/post_install/v5.2.4-post_install.sql]: 1.274 ms

**Comment by Bharath92** [comment]
@avinci @rageshkrishna @himanshu0503 In alpha, there are 4847 projects & 24270 projectAccounts. I have updated the time taken by each query in the PR.

File: migrations/post_install/v5.2.4-post_install.sql (hunk 1 above)
Comment (Bharath92) [inline migrations/post_install/v5.2.4-post_install.sql]: We should probably not do this. 
update projects set "isOrphaned" = true where "id" not in (select distinct "projectId" from "projectAccounts"); which was used previously takes about 30.758 ms to complete.

**Comment by avinci** [comment]
looks good.

**Pull request merged**